parachain/primitives/core/src/outbound.rs

```diff
@@ -16,34 +16,31 @@ pub type FeeAmount = u128;
 pub type GasAmount = u128;
 pub type GasPriceInWei = u128;
 
+/// OutboundFee which covers the cost of execution on both BridgeHub and Ethereum
+#[derive(Copy, Clone, Default, Encode, Decode, PartialEq, Eq, RuntimeDebug, TypeInfo)]
+pub struct OutboundFee<Balance: BalanceT> {
```

Comment:
Lets rename this to `Fees`.

This struct is already in an `outbound` namespace, so no need to have `Outbound` in the name. Also there multiple sub-fees, so lets make the name plural.

Reply:
[Short name](https://github.com/Snowfork/snowbridge/pull/975/commits/f864192282433e928493e657ac2a23800f6af305)